packages/renderer/src/lib/task-manager/TaskManager.svelte

```diff
@@ -21,7 +31,7 @@ function hide() {
 
 function clearCompleted() {
```

Comment:
Should be either renamed or removed and bind the ui directly to clearNotifications()